docs/5___ssd.adoc

```diff
@@ -126,11 +126,37 @@ The following XML child elements are specified for the Connector element:
 |Element |Description
 |Real / Integer / +
 Boolean / String / Enumeration / Binary |Exactly one of these elements *CAN* be present to specify the type of the Connector. See 4.5.1 Type Choice for details.
+|Dimension |One or more of these optional elements define the connector array dimensions, making the connector an array connector. See below for details.
 |ConnectorGeometry |This optional element defines the geometry information of the connector. See below for details.
 |===
 
 The type of the Connector is identified by the presence of one of the XML child elements Real, Integer, Boolean, String, Enumeration, or Binary.
 
+===== Dimension
+
+image:images/image60.png[image,width=295,height=149]
+
+This optional element specifies one dimension of an array connector.
+If no dimension elements are present in a connector, it is a scalar connector.
+The number of dimension elements in a connector provides the dimensionality of the array.
```

Comment:
The number of dimension elements in a connector defines the dimensionality of the array.